source/geometries/Next1EL.cc

```diff
@@ -804,8 +804,7 @@ void Next1EL::BuildFieldCage()
   // WORLD system of reference
   active_position_.set(0.,0.,fieldcage_position_.z()+posz);
 
-  hexrnd_ = new HexagonPointSampler(active_diam_/2., active_length_, 0.,
-				    active_position_);
+  active_gen_ = new PolygonPointSampler(active_diam_/2.,active_diam_/2. + ltube_diam_/2. + tpb_thickn_, active_length_, 6, nullptr, active_position_);
```

Comment:
I think that there's something wrong in the value of the maximum radius in this case: `ltube_diam_` has the same value as `active_diam_`.